iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java

```diff
@@ -39,40 +40,48 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-/** Create regions for specific StorageGroups. */
+/** Create regions for specified Databases. */
 public class CreateRegionGroupsPlan extends ConfigPhysicalPlan {
 
-  // Map<StorageGroupName, List<TRegionReplicaSet>>
+  // Map<Database, List<TRegionReplicaSet>>
   protected final Map<String, List<TRegionReplicaSet>> regionGroupMap;
+  protected final Map<TConsensusGroupId, Long> regionGroupCreateTimeMap;
```

Comment:
Maybe you don't need to maintain a map, just a value? A batch of RegionGroups can use the same time？